bower.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "ember-frost-bunsen",
   "dependencies": {
-    "ember": "2.5.0",
+    "ember": "^2.6.0",
```

Comment:
I don't believe the dependencies in `bower.json` actually get applied to the consumer. I say this because when `perfect-scrollbar` was added to the `bower.json` of `ember-frost-core` consumers were still required to add the dependency, otherwise they'd get an error.